<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="4" hostname="LAPTOP-2PCDKALS" failures="1" tests="4" name="Test(failed)" time="0.058" errors="0" timestamp="2025-07-10T23:09:28 IST">
  <testcase classname="TestNG.TestNg4" name="f" time="0.006">
    <skipped/>
  </testcase> <!-- f -->
  <testcase classname="TestNG.TestNg6" name="z" time="0.002"/>
  <testcase classname="TestNG.TestNg6" name="a" time="0.006">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at TestNG.TestNg6.a(TestNg6.java:16)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- a -->
  <testcase classname="TestNG.TestNg6" name="b" time="0.0">
    <skipped/>
  </testcase> <!-- b -->
  <testcase classname="TestNG.TestNg4" name="c" time="0.0">
    <ignored/>
  </testcase> <!-- c -->
  <testcase classname="TestNG.TestNg4" name="b" time="0.0">
    <ignored/>
  </testcase> <!-- b -->
  <testcase classname="TestNG.TestNg4" name="e" time="0.0">
    <ignored/>
  </testcase> <!-- e -->
  <testcase classname="TestNG.TestNg4" name="d" time="0.0">
    <ignored/>
  </testcase> <!-- d -->
</testsuite> <!-- Test(failed) -->
